src/data/move-attrs/eat-berry-attr.ts

```diff
@@ -1,9 +1,9 @@
 import type { Pokemon } from "#app/field/pokemon";
 import { globalScene } from "#app/global-scene";
-import { BerryModifier, PreserveBerryModifier } from "#app/modifier/modifier";
+import { type BerryModifier, PreserveBerryModifier } from "#app/modifier/modifier";
 import { BooleanHolder } from "#app/utils";
 import { HealFromBerryUseAbAttr } from "#app/data/ab-attrs/heal-from-berry-use-ab-attr";
-import { applyAbAttrs } from "#app/data/ability";
+import { applyAbAttrs } from "../apply-ab-attrs";
```

Comment:
```suggestion
import { applyAbAttrs } from "#app/data/apply-ab-attrs";
```
more examples of import issues, this time in `move-attrs`